docs/contributors/devel/lifted.md

```diff
@@ -0,0 +1,114 @@
+# Overview
+This document explains how lifted code are managed.
+A common user case for this task is developer lifting code from other repository to `pkg/util/lifted` directory.
+
+- [Steps of lifting code](#steps-of-lifting-code)
+- [How to write lifted comments](#how-to-write-lifted-comments)
+- [Examples](#examples)
+
+## Steps of lifting code
+- Copy code from other repository and save it to a go file under `pkg/util/lifted`.
+- Optionally change the lifted code.
+- Add lifted comments for the code, referring to [it](#how-to-write-liftd-comments).
+- run `hack/update-lifted.sh` to update the lifted doc `pkg/util/lifted/doc.go`
+
+## How to write lifted comments
+Lifted comments shall be placed just before the lifted code(may be a func, a type, a var or a const). Except empty line and comments, any others are not allowed between lifted comments and lifted code.
+
+Lifted comments are composed by one or multi comment lines. Each line has format: `+lifted:KEY[=VALUE]`. Value is optional for some key. 
+
+Valid keys are as follow：
+
+- source: 
+
+  Key `source` is requried. Its value indicates where the code is lifted from.
+
+- changed: 
+
+  Key `changed` is optional. If not given, means the lifted code is not changed. Otherwise its value indicates. Value of `changed` is optional, the valid value is `true` or `false`, default to be `true` if not set.
```

Comment:
```suggestion
  Key `changed` is optional. Without it, the lifted code is not changed. Otherwise its value indicates. Value of `changed` is optional (`true` or `false`, defaults to `true`).
```

Reply:
Something is missing in "Otherwise its value indicates."
There should be something following "indicates".
Pls check.